### ISSUE TITLE
New Request: Shortcut to clean all the unread notifications 

### ISSUE THREAD

**Issue by opensamba** [hackedbellini/gnome-shell-notifications-alert#25]
I love your extension, but sometimes I have notification for some messages that I've already read and I don't wont to mix them with the future messages are about to arrive.

So everytime I have to clean the notifications and could be great if a quick shortcut will be implemented.

Thanks

**Comment by hackedbellini** [comment]
@opensamba if I understood your request correctly, I think gnome-shell 3.16 solved that problem for you.

The notifications in it moved to the clock menu, ant there's a "x" that you can press to clear all notifications.

Or is your request something different?

**Comment by opensamba** [comment]
Yes, my request is about to have a shortcut for that x
Is that possible?
Il 12/ott/2015 14:15, "Thiago Bellini" <notifications@github.com> ha
scritto:

**Comment by hackedbellini** [comment]
The shortcut would be for versions < 3.16 or are you saying that the "x" on >= 3.16 isn't "short" enough?

**Comment by opensamba** [comment]
yes, the X is working and is shown, this is not an issue is a feature
request.

Is it possible to associate an hotkey to clear all notification ?
actually I'm doing: *ctrl+m + dx arrow + enter*
and this works in all gnome-shell version

repeat: not an issue and not urgent

thanks

2015-10-12 20:34 GMT+02:00 Thiago Bellini <notifications@github.com>:

**Comment by Anthony25** [comment]
Hello,
@opensamba, maybe @hackedbellini will not be agree with me :p, but I do not see how it is related to this addon. You should do your request directly to the Gnome team, no ?
